packages/gamut/src/NotificationListNew/EmptyNotification.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { FlexBox, Text } from '..';
+import {
+  emptyNotificationMessages,
+  slugToSvgMap,
+} from './assets/emptyNotificationMessages';
+
+export const EmptyNotification = () => {
+  const getRandomIndex = (max: number) =>
+    Math.floor(Math.random() * Math.floor(max));
+  const randomIndex = getRandomIndex(emptyNotificationMessages.length);
```

Comment:
✨   https://lodash.com/docs/4.17.15#sample ✨ 

```ts
import { sample } from 'lodash';
```

```suggestion
  const message = sample(emptyNotificationMessages);
```

Reply:
✨✨✨